packages/web-frontend/src/tests/components/dateRangePicker.test.js

```diff
@@ -23,8 +27,8 @@ describe('dateRangePicker', () => {
       render(<DateRangePicker granularity={key} />);
 
       const labelText = screen.getByLabelText('active-date');
-      const startDate = MIN_MOMENT_DATE.format(value.rangeFormat);
-      const endDate = MAX_MOMENT_DATE.startOf(value.momentUnit).format(value.rangeFormat);
+      const startDate = formatMomentAsString(MIN_MOMENT_DATE, key, value.rangeFormat);
```

Comment:
Is it ok to use this util as the util itself is not being tested here?

Reply:
It's actually a better practice to use the end value directly

Reply:
Should I create a map up the top?

```javascript
MIN_MOMENT_STRINGS = {
  [GRANULARITIES.SINGLE_DAY]: '20 Jan 2020',
  ...
};
```
Then 
```javascript
const startDate = MIN_MOMENT_STRINGS[key];
```

Reply:
Yeap sounds good!